internal/mode/static/nginx/runtime/manager_test.go

```diff
@@ -7,26 +7,152 @@ import (
 	"testing"
 	"time"
 
-	"github.com/go-logr/logr"
+	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/nginx/runtime"
+	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/nginx/runtime/runtimefakes"
```

Comment:
Separate these internal imports into a third category below the third-party imports.

Reply:
will do